pallets/pallet-lottery/src/staking/deposit_strategies.rs

```diff
@@ -40,12 +40,45 @@ pub(super) fn reactivate_bottom_collators<T: Config>(
         let staked = StakedCollators::<T>::get(collator.clone());
         let info = pallet_parachain_staking::Pallet::<T>::candidate_info(collator.clone())
             .expect("is active collator, therefore it has collator info. qed");
+
+        // If collator not exist in delegatorState(PotAccount).delegations, ignore
```

Comment:
Not sure why use `crate::Pallet::<T>::account_id()` as delegator address.
`delegator_state` is for showing who the delegator is delegating for.

Reply:
user deposit to lottery pool, lottery pool use Pallet account as a delegator to staking module. as many user deposit to the pool, the Pallet account may stake to multiple collator.

Reply:
Got it